pytensor/tensor/slinalg.py

```diff
@@ -131,6 +146,12 @@ def conjugate_solve_triangular(outer, inner):
         else:
             return [grad]
 
+    def try_inplace_inputs(self, candidate_inputs: list[int]) -> "Op":
+        if candidate_inputs == [0]:
```

Comment:
As per the signature this should not return None. Maybe that should be the default? Or return self (so it's still an Op)?

In the rewrite we're not expecting None to be returned, but only handling the NotImplementedError. I think there would be an error if None is returned as we next try to access `.destroy_map` property